probe/endpoint/reporter.go

```diff
@@ -214,3 +235,29 @@ func (r *Reporter) makeEndpointNode(namespaceID string, addr string, port uint16
 func newu64(i uint64) *uint64 {
 	return &i
 }
+
+// procParsingSwitcher make sure that if eBPF tracking is enabled,
+// connections coming from /proc parsing are only walked once.
+func (r *Reporter) procParsingSwitcher() {
+	if r.conf.WalkProc && r.conf.UseEbpfConn {
+		r.conf.WalkProc = false
+		r.ebpfTracker.initialize()
```

Comment:
Shouldn't this be part of `newEbpfTracker`? The initialization should happen in a single place.